test/acceptance/todo-cli-test.js

```diff
@@ -5,7 +5,7 @@ import { differenceInDays, subDays } from 'date-fns';
 import { setupProject, teardownProject, runBin } from '../helpers/bin-tester.js';
 import setupEnvVar from '../helpers/setup-env-var.js';
 
-jest.setTimeout(10_000);
+jest.setTimeout(100_000);
```

Comment:
Perhaps we should be a bit more incremental? A bit worried if we have tests taking 100 seconds...
```suggestion
jest.setTimeout(20_000);
```